rpm/0002-dict-Add-fallback-if-getentropy-is-not-supported.patch

```diff
@@ -8,11 +8,11 @@ Subject: [PATCH] dict: Add fallback if getentropy is not supported
  1 file changed, 12 insertions(+)
 
 diff --git a/dict.c b/dict.c
-index 49e1c6bf..0d31233f 100644
+index a238cd9d03d3e1ea286e38417f17d83a52e536c3..a323a9e3ac47edf1716b51438c831ba9c119ada1 100644
 --- a/dict.c
 +++ b/dict.c
-@@ -970,6 +970,18 @@ xmlInitRandom(void) {
-             if (getentropy(globalRngState, sizeof(globalRngState)) == 0)
+@@ -983,6 +983,18 @@ xmlInitRandom(void) {
+             if (errno == ENOSYS)
                  break;
  
 +            if (errno == ENOSYS) {
```

Comment:
This branch is now unreachable.  `getentropy` check now happens before the patch, so maybe the upper `if` here can just be removed?

Reply:
Fixed